doc/usersguide/tools.rst

```diff
@@ -0,0 +1,104 @@
+DAGMC tools
+===========
+
+The DAGMC infrastructure has a number of tools that are used to process, fix,
+diagnose and repair processed models. This document serves to better inform and
+educate what those tools are and how to use them.
+
+..  _make_watertight:
+
+make_watertight
+~~~~~~~~~~~~~~~
+
+A model is considered watertight if the faceting of all topologically linked
+surfaces are coincident. Models produced by Cubit/Trelis are not guarenteed to
+be watertight, and as a general rule, the more complicated the model, the less
+likely it is to be completely watertight.
+
+The ``make_watertight`` tool aims to fix any watertightness issues in a model by
+using faceted curve information to seal triangle facets that meet on the same
+curve. An example of its use is shown here: the red lines show the geometry
+before ``make_watertight`` and the black lines show the geometry after its use.
+
+..  image:: watertight.png
```

Comment:
The zoom box in this image https://svalinn.github.io/DAGMC/_images/watertight.png is misaligned

Reply:
Do we have the original image file anywhere for this? Was this maybe @pshriwise's original creation?

Reply:
Or maybe @makeclean has the original?

Alternative to regenerating this misaligned image is to just use this image in its place which demonstrates the make_watertight concept too:
![sealed-facets](https://user-images.githubusercontent.com/4060017/77079775-80965b80-69c6-11ea-929a-37002f863dee.png)

Reply:
Image has been fixed by @gonuke